### PR TITLE
Spell Check in config values.yaml

### PR DESCRIPTION
**Pull request by nhalstead**
## Proposed changes

Documentation changes. Non-breaking.
This just corrects the spelling of some words in the `values.yaml` file.

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](https://github.com/splunk/splunk-connect-for-kubernetes/blob/develop/CONTRIBUTING.md) doc
- [x] I have read the [CLA](https://github.com/splunk/splunk-connect-for-kubernetes/blob/develop/CLA.md)
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: helm-chart/splunk-connect-for-kubernetes/values.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Splunk Connect for Kubernetes is a umbraller chart for three charts
+# Splunk Connect for Kubernetes is a umbralla chart for three charts
```
Comment (sigbjornt) [inline helm-chart/splunk-connect-for-kubernetes/values.yaml:1]: Is it not "umbrella"

Comment (rockb1017) [reply helm-chart/splunk-connect-for-kubernetes/values.yaml:1]: Yea, please update. Thank you

**Review by rockb1017 — CHANGES_REQUESTED** [review]
Thank you. just one "umbrella" and others looks good!

File: helm-chart/splunk-connect-for-kubernetes/values.yaml (hunk 1 above)
Comment (nhalstead) [reply helm-chart/splunk-connect-for-kubernetes/values.yaml]: FIxed.

**Review by rockb1017 — APPROVED** [review]

**Pull request merged**